game/songs.cc

```diff
@@ -94,18 +95,25 @@ void Songs::reload_internal() {
 	Paths systemSongs = getPathsConfig("paths/system-songs");
 	Paths paths = getPathsConfig("paths/songs");
 	paths.insert(paths.begin(), systemSongs.begin(), systemSongs.end());
-
+	std::vector<std::future<void>> futures;
 	for (auto it = paths.begin(); m_loading && it != paths.end(); ++it) { //loop through stored directories from config
```

Comment:
Use ranged based for.

Reply:
Fixed, kept it the same as previously before but ranged based loops are better readable yes.